crates/nu-test-support/src/lib.rs

```diff
@@ -22,8 +25,8 @@ pub const NATIVE_PATH_ENV_SEPARATOR: char = ';';
 pub const NATIVE_PATH_ENV_SEPARATOR: char = ':';
 
 impl Outcome {
-    pub fn new(out: String, err: String) -> Outcome {
-        Outcome { out, err }
+    pub fn new(out: String, err: String, status: ExitStatus) -> Outcome {
+        Outcome { out, err, status }
```

Comment:
In general a good idea to track this as well.

Would you mind forking out the changes to the `nu!` macro test infrastructure into a separate PR? 

I think just for the discoverability of the potential additions it would be helpful to be able to point folks to that.